docs/q-and-a.md

```diff
@@ -6,8 +6,14 @@ The Q&A frontend is a finder-frontend feature that walks users through the facet
 
 Any, all or no filters can be selected for each facet, and any question may be skipped. At the end of the process, the user is directed to the given finder with their selections intact.
 
-At the time of writing, this feature has been defined for one finder and relies on a [YAML file](../lib/prepare_business_uk_leaving_eu.yaml). This file defines things like the base path of the Q&A and the underlying finder, as well as questions, question types and descriptions for each question page.
+It emulates the look and feel of a [Smart Answer](https://github.com/alphagov/smart-answers), but the code lives here so that we can use the user's answers in pre-populating the finder.
 
-The YAML file and finder content item together provide all the data required for the Q&A frontend.
+This [feature has been defined for one finder](https://github.com/alphagov/finder-frontend/pull/675) and relies on a [YAML file](../lib/prepare_business_uk_leaving_eu.yaml). This file defines things like the base path of the Q&A and the underlying finder, as well as questions, question types and descriptions for each question page.
```

Comment:
I think you can remove the link to the PR.  Also, you could there is only one finder that uses the feature and link to it's start page, e.g. https://www.gov.uk/business-uk-leaving-eu.

Reply:
The information in this statement "This file defines things like the base path of the Q&A and the underlying finder, as well as questions, question types and descriptions for each question page." Seems to be repeated below in the bullet points, so I think it can be removed.